src/js/actions/layers.js

```diff
@@ -482,6 +482,13 @@ define(function (require, exports) {
             return Promise.resolve();
         }
 
+        var upToDateDocument = this.flux.store("document").getDocument(document.id);
```

Comment:
It sure would be nice to just change the first argument of `resetLayers` to be documentID.  Another day I guess.  Something about `upToDateDocument` rubs me the wrong way, as a variable name.